app/controllers/home_controller.rb

```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
   def show
+    #passing this variable to the views?
     @client_id = ENV["client_id"]
```

Comment:
Thanks @s-espinosa! That answers my question. I'm trying to make WIP pull requests that address smaller questions and even stylistic points; a micro PR strategy. I want to hand roll OAuth for this project, but it's good to know that OmniAuth hides the ```client_id``` in the HTML.